templates/ImageBuilder/cis-for-eb-image-pipeline-template.yaml

```diff
@@ -15,6 +15,12 @@ Parameters:
     Default: 50
     MinValue: 8
     MaxValue: 500
+  ImageLifecycleMonths:
+    Description: Number of months for images to exist before being deleted
+    Type: Number
+    Default: 6
```

Comment:
seems ok, just wondering what the rational was for this value.  or is it just a place holder and you plan to override this value when using this template?

Reply:
There's no particular rationale other than it's suitable for the release cadence of the CIS image and our use of Elastic Beanstalk.